packages/tsconfig-reference/copy/pt/options/importsNotUsedAsValues.md

```diff
@@ -0,0 +1,15 @@
+---
+display: "Importações não utilizadas como valores"
+oneline: "faz algo"
```

Comment:
@orta I think the English version needs tweaking

Reply:
haha, so it does

Reply:
`oneline: "Controls which syntax you use for importing types"`